tests/sqllogictests/suites/query/02_function/02_0012_function_datetimes_tz

```diff
@@ -22,7 +22,7 @@ set timezone='Asia/Shanghai'
 query T
 select to_timestamp(1630320462000000)
 ----
-2021-08-30 18:47:42.000000
+2021-08-31 02:47:42.000000
```

Comment:
this is wrong!

Reply:
How about support this as TO_TIMESTAMP_LTZ?

If to_timestamp(int) consider tz, I think this is TO_TIMESTAMP_LTZ in sf.

Reply:
micro-seconds has no business with Time Zone.

Reply:
Fix